docs/_data/nav.yml

```diff
@@ -0,0 +1,46 @@
+- title: "Get Started"
+  subcategories:
+  - subtitle: "Quick start tutorial"
+    subhref: "/soda-sql/getting-started/5_min_tutorial"
+  - subtitle: "Install Soda SQL"
+    subhref: "/soda-sql/getting-started/installation"
+  - subtitle: "Configure Soda SQL"
+    subhref: "/soda-sql/getting-started/configure"
+
+- title: "Documentation"
+  subcategories:
+  - subtitle: "How Soda SQL works"
+    subhref: "/soda-sql/documentation/concepts"
+  - subtitle: "Soda SQL CLI commands"
+    subhref: "/soda-sql/documentation/cli"
+  - subtitle: "Warehouse YAML"
+    subhref: "/soda-sql/documentation/warehouse"
+  - subtitle: "Set warehouse configuration"
+    subhref: "/soda-sql/documentation/warehouse_types"
+  - subtitle: "Scan YAML"
+    subhref: "/soda-sql/documentation/scan"
+  - subtitle: "Metrics"
+    subhref: "/soda-sql/documentation/metrics"
+  - subtitle: "Tests"
+    subhref: "/soda-sql/documentation/tests"
+  - subtitle: "Apply filters"
+    subhref: "/soda-sql/documentation/filtering"
+  - subtitle: "Configure programmatic scans"
+    subhref: "/soda-sql/documentation/programmatic_scan"
+  - subtitle: "Configure orchestrated scans"
+    subhref: "/soda-sql/documentation/orchestrate_scans"
+  - subtitle: "Soda Cloud Architecture"
+    subhref: "/soda-sql/documentation/soda-cloud-architecture"
+  - subtitle: "Connect to Soda Cloud"
+    subhref: "/soda-sql/documentation/connect_to_cloud"
+  - subtitle: "Create monitors and alerts"
+    subhref: "/soda-sql/documentation/monitors"
+  - subtitle: "Integrate with Slack"
+    subhref: "/soda-sql/documentation/integrate-slack"
+  - subtitle: "Troubleshoot"
+    subhref: "/soda-sql/documentation/troubleshoot"
+  - subtitle: "Glossary"
+    subhref: "/soda-sql/documentation/glossary"
+
+- title: "Community"
+  href: "/soda-sql/community"
```

Comment:
You might want to change your editor to add empty new lines at the end of the file :)